R/location.R

```diff
@@ -132,6 +116,44 @@ orderly_location_add <- function(name, type, args, root = NULL, locate = TRUE) {
 }
 
 
+##' @rdname orderly_location_add
+##'
+##' @param path The path to the other archive root. This should
+##'   generally be an absolute path, or the behaviour of outpack will
+##'   be unreliable.
+orderly_location_add_path <- function(name, path, root = NULL) {
+  args <- list(path = path)
+  orderly_location_add(name, "path", args, root = root)
+}
+
+
+##' @rdname orderly_location_add
+##'
+##' @param url The location of the server, including protocol, for
+##'   example `http://example.com:8080`
+orderly_location_add_http <- function(name, url, root = NULL) {
+  args <- list(url = url)
+  orderly_location_add(name, "http", args, root = root)
+}
+
+
+##' @rdname orderly_location_add
+##'
+##' @param token The value for your your login token (currently this
+##'   is a GitHub token with `read:org` scope). If `NULL`, orderly2
+##'   will perform an interactive authentication against GitHub to
+##'   obtain one.
```

Comment:
Technically this can be a Packit JWT instead of a GitHub PAT, but we don't really publicize any way of obtaining those so I think it is fine if you don't want to mention it here.

This is the only place a JWT would be used today: https://github.com/mrc-ide/orderly-action/blob/main/packit-login.R#L65-L69

Reply:
I just copied this in from the string that was present in the Details for now